alacritty/src/event.rs

```diff
@@ -999,8 +1007,11 @@ impl<N: Notify + OnResize> Processor<N> {
     }
 
     /// Run the event loop.
-    pub fn run<T>(&mut self, terminal: Arc<FairMutex<Term<T>>>, mut event_loop: EventLoop<Event>)
-    where
+    pub fn run<T>(
+        &mut self,
+        terminal_ptr: Arc<FairMutex<Term<T>>>,
```

Comment:
What is going on here? Why did you rename these things?

Reply:
I renamed it because I need the `Arc` pointer for the action context (used by the `Action::Command` handler later) but [this line](https://github.com/alacritty/alacritty/blob/17923efccfb7e6bbcdeb5f748796d9ee1ea70149/alacritty/src/event.rs#L1066) redefines the variable.